src/ml/neural_net/style_transfer/mps_style_transfer.m

```diff
@@ -44,30 +44,25 @@ @interface TCMPSStyleTransfer()
 @end
 
 @implementation TCMPSStyleTransfer
-- (instancetype) initWithParameters:(NSDictionary<NSString *, NSData *> *)weights
-                          numStyles:(NSUInteger)numStyles {
+
+- (instancetype) initWithDev:(id<MTLDevice>) dev
+                commandQueue:(id<MTLCommandQueue>) commandQueue
+                     weights:(NSDictionary<NSString *, NSData *> *) weights
+                   numStyles:(NSUInteger) numStyles {
   self = [super init];
   if (self) {
-    // Set Default parameters values
+    // Allocate a _dev and _commandQueue
```

Comment:
Nit: You're not actually allocating anything here. Allocation typically implies new memory is being appropriated from the system